### DISCUSSION TITLE
Disable HTTPS for Master Realm

### DISCUSSION THREAD

**Discussion by awspd** [keycloak/keycloak#28067] [Q&A]
Hi everybody, I recently built out KeyCloak on an AWS EC2 instance (Windows Server 2022) w/ Java (JRE) and SE using instructions here.
https://medium.com/@awosolat/how-to-install-keycloak-17-on-windows-machine-b2830503751f
I since attempted to configure HTTPS and knew I'd lock myself out but proceeded anyway to enforce HTTPS. Now I cannot seem to access the console. Does anybody know how I can reset the SSL parameter to not be enforced on a Windows box? The majority of threads I've reviewed utilize .SH (bash) or relate to Linux distros which won't fix my issue! Thank you!!

**Comment by codespearhead** [comment]
Sounds like an XY problem [1].

I have a feeling you're running it behind a reverse proxy, in which case I suggest doing as this answer [2][3] says.

***

[1] https://xyproblem.info/
[2] https://github.com/keycloak/keycloak/discussions/28279
[3] https://github.com/keycloak/keycloak/discussions/28279#discussioncomment-9001818
